### DISCUSSION TITLE
Output: mount.nfs: mount system call failed

### DISCUSSION THREAD

**Discussion by linpanru** [kubernetes-csi/csi-driver-nfs#423] [Q&A]
![image](https://user-images.githubusercontent.com/56077231/223033148-b22bb65c-e8d9-4809-96d3-50be681ec5b6.png)

i confirm nfs-server is ready ,my csi-driver-nfs version is v4.1.0 , and my system is arm , i need help ,god

**Comment by andyzhangx** [comment]
what's the result if you run "mount -t nfs ..." on the k8s node directly?

**Reply by linpanru** [reply]
> what's the result if you run "mount -t nfs ..." on the k8s node directly?

![image](https://user-images.githubusercontent.com/56077231/223044237-b55374e6-f164-42af-8071-e9ef6293c283.png)
on the k8s node ,when i use nfs server svc address for mount , it will report this eroor : Name or service not known ;  but when i use nfs server pod IP to mount ,it is ok

**Reply by andyzhangx** [reply]
so it be related to this setting? Are you able to change the value to check whether `ClusterFirst` setting works?
https://github.com/kubernetes-csi/csi-driver-nfs/blob/7e59f2a38d551a92d23bb0d1f4540c16b697b3a3/deploy/csi-nfs-node.yaml#L21

**Reply by linpanru** [reply]
> so it be related to this setting? Are you able to change the value to check whether `ClusterFirst` setting works?
> 
> https://github.com/kubernetes-csi/csi-driver-nfs/blob/7e59f2a38d551a92d23bb0d1f4540c16b697b3a3/deploy/csi-nfs-node.yaml#L21

![image](https://user-images.githubusercontent.com/56077231/223046402-e1004793-d8e0-47a6-9ad5-6a36bde19afb.png)
my csi-nfs-node & csi-nfs-controller dnsPolicy is ClusterFirstWithHostNet and hostNetwork is true ;  on X86 environment  it is ok , but on Arm environment , It was fine at first，but when i change nfs-server to another k8s node ，i use nfs server svc addresss to mount ,it does not work.
